modules/manually-configure-iam-nutanix.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_nutanix/configuring-iam-nutanix.adoc
+
+:_content-type: PROCEDURE
+[id="manually-create-iam-nutanix_{context}"]
+= Configuring IAM for Nutanix
+
+Installing the cluster requires that the Cloud Credential Operator (CCO) operate in manual mode. While the installation program configures the CCO for manual mode, you must specify the identity and access management secrets.
+
+.Prerequisites
+
+* You have configured the `ccoctl` binary.
+* You have an existing `install-config.yaml` file.
+* You have set the following environment variables:
++
+`NUTANIX_ENDPOINT`:: Specifies the endpoint for Prism Central.
+`Nutanix_PORT`:: Specifies the port that is used to log into Prism Central.
+`NUTANIX_USER`:: Specifies the username that is used to log into Prism Central.
+`NUTANIX_PASSWORD`:: Specifies the password that is used to log into Prism Central.
+
+.Procedure
+
```

Comment:
A demo of this procedure would be beneficial. I believe some of the annotation values can be a bit crisper.

Based this section on the CCO doc [here](https://github.com/openshift/cloud-credential-operator/blob/master/docs/ccoctl.md#nutanix).